nvdaHelper/vbufBackends/mshtml/node.cpp

```diff
@@ -355,12 +371,12 @@ MshtmlVBufStorage_controlFieldNode_t::~MshtmlVBufStorage_controlFieldNode_t() {
 void MshtmlVBufStorage_controlFieldNode_t::preProcessLiveRegion(const MshtmlVBufStorage_controlFieldNode_t* parent, const std::map<std::wstring,std::wstring>& attribsMap) {
 	auto i=attribsMap.find(L"HTMLAttrib::aria-live");
 	if(i!=attribsMap.end()&&!i->second.empty()) {
-		this->ariaLiveNode=((i->second.compare(L"polite")==0)||(i->second.compare(L"assertive")==0))?this:NULL;
+		this->ariaLiveNode = ((i->second.compare(L"polite") == 0) || (i->second.compare(L"assertive") == 0)) ? this : nullptr;

```

Comment:
I think the readability of this could be improved by using operator == and naming the condition. What does the condition represent, "not off"?

```suggestion
		bool isAriaLiveEnabled = i->second == L"polite" || i->second == L"assertive";
		this->ariaLiveNode = isAriaLiveEnabled ? this : nullptr;
```

Also, this an opportunity to use a better name than `i`, like `ariaLiveAttribute`